concepts/models.py

```diff
@@ -48,16 +48,19 @@ class Concept(HeritableObject):
     REJECTED = 'Rejected'
     RESOLVED = 'Resolved'
     MERGED = 'Merged'
+    FLAGGED = 'Flagged'
     concept_state_choices=  (
         (PENDING, 'Pending'),
         (REJECTED, 'Rejected'),
         (RESOLVED, 'Resolved'),
         (MERGED, 'Merged'),
+        (FLAGGED, 'Flagged'),
     )
     concept_state=models.CharField(max_length=10, choices=concept_state_choices,
                                    default='Pending')
     merged_with = models.ForeignKey('Concept', related_name='merged_concepts',
                                     **optional, on_delete=models.CASCADE)
+    comment = models.TextField(blank=True, null=True)
```

Comment:
This seems to narrow, this should probably a one to many relationship, so the user can leave more than one comment. At this point, it's fine to have only create a comment when flagging a concept, but I'll create a ticket so that other users can respond to comments.

Reply:
What I meant is that the variable here should be able to accommodate multiple comments even though only one needs to be created for this ticket.